installers/Launcher.hs

```diff
@@ -39,8 +39,12 @@ launcherArgs launcher = unwords $
         "--kademlia-peers-file", "ip-dht-mappings",
         "--system-start", "1499360281",
         "--wallet",
+        "--tlscert", quote (runtimePath launcher <> tlsPath <> "server" <> pathSeparator <> "server.crt"),
```

Comment:
@deepfire , then I should probably remove one extra `tls` dir. Because `tlsPath = "tls/tls/` . I will set it to `tlsPath = "tls/"` based on your input. Ty